connector/connect/client/jvm/src/test/scala/org/apache/spark/sql/connect/client/util/RemoteSparkSession.scala

```diff
@@ -49,7 +49,7 @@ import org.apache.spark.util.Utils
 object SparkConnectServerUtils {
 
   // Server port
-  private[connect] val port = ConnectCommon.CONNECT_GRPC_BINDING_PORT + util.Random.nextInt(1000)
+  val port: Int = ConnectCommon.CONNECT_GRPC_BINDING_PORT + util.Random.nextInt(1000)
```

Comment:
??? Why this change?

Reply:
https://github.com/apache/spark/pull/40684/files/f3d1eade9f90517039842fa59815aaf46f0a53b3#diff-7fa161b193c8792c8c0d8dd4bcae3e683ab8553edafa2ae5c13df42b26f612b0R37
just want to use it in `ClientE2ETestSuite` to create another SparkSession. Any better idea?

Reply:
You could use `private[spark] val port: Int` accessor?